orchagent/macsecorch.cpp

```diff
@@ -1275,6 +1275,8 @@ bool MACsecOrch::createMACsecPort(
             phy);
     });
 
+    m_port_orch->setMACsecEnabledState(port_id, true);
+
```

Comment:
Since we cannot guarantee that the macsec on a Port will be enabled before the MTU is set, I feel we need to set the MTU explicitly here. 

Or another way will be to check for macsec profile present in the port in config table and update the MTU accordingly in the PortsOrch::setPortMtu API.

Reply:
We will always update the MTU value in this function. Meanwhile, in the setPortMTU function, we will also check whether the MACsec enabled. So, I don't think there is any issue if enabling MACsec comes before the MTU setting.